pages/knn-classification.py

```diff
@@ -5,17 +5,19 @@
 
 dash.register_page(
     __name__,
-    description="Visualize scikit-learn's k-Nearest Neighbors (kNN) classification",
+    description="Visualize scikit-learn's k-Nearest Neighbors (kNN) classification. It has a top-bottom layout and a regular-callback.",
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
 
-#### Plotly Documentation:  
+#### Dash Components in App:
+- [Slider](https://dash.plotly.com/dash-core-components/slider)
 
-- [Visualize scikit-learn's k-Nearest Neighbors (kNN) classification](https://plotly.com/python/knn-classification/)
```

Comment:
Let's leave this first link under the Plotly documentation but above the link you added to the scatter plot; this [link](https://plotly.com/python/knn-classification/) is relevant because this app is displayed on the page of that link.

Reply:
Done